src/App.js

```diff
@@ -5,17 +5,20 @@ import AddTodoForm from './AddTodoForm';
 
 function App() {
 
-  const [newTodo, setNewTodo] = React.useState('');
+  const [todoList, setTodoList] = React.useState([]);
+  const addTodo = (newTodo) => {
+    setTodoList([...todoList, newTodo]);
+  }
 
   return (
     <div>
       <h1>Todo List</h1>
-      <AddTodoForm onAddTodo={setNewTodo} />
+      <AddTodoForm onAddTodo={addTodo} />
+      {/* is this addTodo below doing anything? Why is it here? ask during mentor session*/}
       <p>
-        {newTodo}
+        {addTodo}
```

Comment:
This `addTodo` is not doing anything for you except triggering an interesting error in your console. See if you can spot it. What are your guesses as to what you should do? Also, double-check the instructions.

Let's try collaborating through github for this issue. Sometimes this is quicker and much easier than setting up a meeting. You can respond directly in this thread while we figure it out.

Always happy to meet for a call but this is a necessary workflow technique that you need to get comfortable with in a professional environment.

Reply:
After 1:1 with Sergey, I removed this section, but  didn't push the changes.